sev-types/src/command/code.rs

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! The `code` module contains convenient symbolic descriptions
+//! of the possible commands that may be made to the SEV platform.
+//!
+//! Chapter 4.3, "Command Identifiers"
+
+use enumerate::enumerate;
+
+enumerate! {
```

Comment:
```suggestion
enumerate::enumerate! {
```

Then remove the `use` statement above.